multigeiger/display.h

```diff
@@ -4,8 +4,60 @@
 #define _DISPLAY_H_
 
 void setup_display(bool loraHardware);
-void DisplayGMC(int TimeSec, int RadNSvph, int CPM, bool use_display, bool connected, bool ble_active);
+void DisplayGMC(int TimeSec, int RadNSvph, int CPM, bool use_display);
 void clearDisplayLine(int line);
 void displayStatusLine(String txt);
 
+// supported status indexes and values:
+
+#define ST_NODISPLAY 0  // this is used for all subsystems
+
+#define STATUS_WIFI 0
+#define ST_WIFI_OFF 1
+#define ST_WIFI_CONNECTING 2
+#define ST_WIFI_CONNECTED 3
+#define ST_WIFI_ERROR 4
+
+#define STATUS_BT 1
+#define ST_BT_OFF 1
+#define ST_BT_CONNECTING 2
+#define ST_BT_CONNECTED 3
+#define ST_BT_ERROR 4
+
+#define STATUS_LORA 2
+#define ST_LORA_OFF 1
+#define ST_LORA_CONNECTING 2
+#define ST_LORA_CONNECTED 3
+#define ST_LORA_ERROR 4
+
+#define STATUS_HV 3
+#define ST_HV_OK 1
+#define ST_HV_ERROR 2
+
+#define STATUS_SCOMM 4
+#define ST_SCOMM_IDLE 1
+#define ST_SCOMM_SENDING 2
+#define ST_SCOMM_ERROR 3
+
+#define STATUS_MADAVI 5
+#define ST_MADAVI_IDLE 1
+#define ST_MADAVI_SENDING 2
+#define ST_MADAVI_ERROR 3
+
+#define STATUS_TTN 6
+#define ST_TTN_IDLE 1
+#define ST_TTN_SENDING 2
+#define ST_TTN_ERROR 3
+
+#define STATUS_TOILET 7
+#define ST_TOILET_IDLE 1
+#define ST_TOILET_SENDING 2
+#define ST_TOILET_ERROR 3
+
```

Comment:
please check which make sense and which don't.

Reply:
s. above, maybe the status constants can be merged that way.

Reply:
maybe they even need to be less consistent than now.

e.g. i am not sure whether lora really has a connecting / connected state or whether it is rather like udp "just send the stuff".